PoW-vs-Pos/en/en_transcription.md

```diff
@@ -0,0 +1,102 @@
+
+Transcript:
+
+(00:02) in the winter of 2009 a piece of software was released on the internet digital money called bitcoin like throwing water on burning oil it ignited and this is the biggest story of this decade bitcoin sent shockwaves through the world spreading like wildfire and disrupting everything it touched nobody knew what it was nobody knew how to value it and nobody could control it and like any new technology it came with pushback the whole bitcoin market is about a billion dollars it's a novelty for the geek squad on top of all
```

Comment:
Distinguere le frasi non del narratore `digital money called Bitcoin`, `and this is the biggest story of this decade` e `the whole bitcoin market is about a billion dollars it's a novelty for the geek squad`